environment.yml

```diff
@@ -7,3 +7,4 @@ dependencies:
   - sphinx=5.3.0
   - sphinxcontrib-mermaid=0.8.1
   - sphinx_rtd_theme=1.1.1
+  - sphinx-jsonschema=1.19.1
```

Comment:
I don't get jsonschema to work. It still shows `.. jsonschema:: filename.json` instead of the content...

Reply:
> Maybe an unexpected interaction with myst parser?

I commented the myst parser (and mermaid), but to no avail.

> Is the file path correct?

Yes

> Is the extension really loaded (such that the directive gets acted upon)?

At least it does not show any errors locally or in the CI.

Reply:
I got it to work:
In an markdown file you have to use a colon fence:
~~~
:::{jsonschema}
your schema
:::
~~~


This does not work, however, for referencing files, only for showing a code snippet.

It does not render our schemata nicely 😭  (it does not render anything at all, as it does not recognize the keywords...).

Reply:
well, progress I guess? :-/

Reply:
I went with the data-viewer, that works better (but does not allow examples...).